lisp/ess-sp4-d.el

```diff
@@ -34,11 +34,10 @@
 
```

Comment:
I read on the mailing list that people still use old versions of S-plus because you have to pay to buy new ones. I have no idea if someone is still stuck with S-plus and uses ESS though. We're almost in 2020, at some point S-plus has got to die :)